plugins/k8smeta/README.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Audit Events Plugin
+
+## Introduction
+
+This plugin enriches Falco syscall flow with Kubernetes Metadata coming from the API server.
+The plugin uses a GRPC channel to communicate with a remote [collector](https://github.com/falcosecurity/k8s-metacollector). The collector is indipendent from the plugin and should be deployed as a separate component. The main role of the plugin is to associate each syscall with information about the pod in which they are thrown.
+
+### Functionality
+
+TODO
+
+## Capabilities
+
+The `k8smeta` plugin implements these capabilities:
+* `extraction`
+* `parsing`
+* `async`
+
+### Supported Fields
+
+<!-- README-PLUGIN-FIELDS -->
+|            NAME            |      TYPE       |      ARG      |                                                                                                                                                 DESCRIPTION                                                                                                                                                  |
+|----------------------------|-----------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `k8sres.pod.name`          | `string`        | None          | Kubernetes pod name.                                                                                                                                                                                                                                                                                         |
+| `k8sres.pod.id`            | `string`        | None          | Kubernetes pod ID.                                                                                                                                                                                                                                                                                           |
```

Comment:
Update as well to `k8sres.pod.uid`. In general, let;s sync docs between here and the `k8s` fields extracted from the container runtime, see also https://github.com/falcosecurity/libs/pull/1575


During today's maintainer meeting I also didn't quite get why we choose the prefix `k8sres.` -- I think `k8smeta.` would be so much better and make the most sense.

Reply:
Additional note for blog post / documentation: Explain very detailed how this differs from the `k8s` fields and when this would be needed or why you should consider this plugin in cases where the `k8s` fields extracted from the runtime socket are all you need.

Reply:
thank you for the suggestions about the blog/docs!

Reply:
> Update as well to k8sres.pod.uid. In general, let;s sync docs between here and the k8s fields extracted from the container runtime, see also https://github.com/falcosecurity/libs/pull/1575

I agree, i will change it in `*.pod.uid` and same for all others resources, like `namespace.uid`

Reply:
> During today's maintainer meeting I also didn't quite get why we choose the prefix k8sres. -- I think k8smeta. would be so much better and make the most sense.

This is a valid point, I don't like too much the name `k8sres`. The idea we talked about with Leo is to express with the prefix what the filter check will extract. in this case `k8sres` means that we will extract a `k8s` resource, probably something like `k8sresource` would be more self-explanatory. BTW `k8smeta` could be a valid name, first is the same name as the plugin and this in some way is coherent and unique, and its meaning could be in line with what we want, `k8smeta` could mean we will extract a `k8s metadata`. It is also in line with the name of the remote collector, it is called `k8s-metacollector`(https://github.com/falcosecurity/k8s-metacollector). WDYT? @leogr

Reply:
I agree that `k8smeta` would be a more suitable name.

Reply:
`k8smeta` is ok for me too.

Reply:
solved both points